src/index.tsx

```diff
@@ -132,16 +134,14 @@ export class Blurhash extends React.PureComponent<BlurhashProps> {
 				{...this.props}
 				onLoadStart={this._onLoadStart}
 				onLoadEnd={this._onLoadEnd}
-				// @ts-expect-error
 				onLoadError={this._onLoadError}
 			/>
 		);
 	}
 }
 
-// requireNativeComponent automatically resolves 'BlurhashView' to 'BlurhashViewManager'
-const NativeBlurhashView = requireNativeComponent<BlurhashProps>(
-	'BlurhashView',
-	// @ts-expect-error this second argument is still not public, but probably required for TurboModules.
-	Blurhash,
-);
+const NativeBlurhashView = Platform.OS === 'web' ? require('./web/blurhashView').default : requireNativeComponent<BlurhashProps>(
```

Comment:
I tried a couple iterations of this.

Setting `NativeBlurhashView` as a `let` then assigning within an if statement, caused issues with Typescript.  Didn't like that it was unset for a short period.

Using `Platform.select` still caused `reuireNativeComponent` to be called on web.  In `react-native`, irrelevant `Platform.select` branches are removed at build time - doesn't look like that happens with `react-native-web`, unfortunately.